app/plugin_install.go

```diff
@@ -17,11 +17,11 @@ import (
 )
 
 // InstallPlugin unpacks and installs a plugin but does not enable or activate it.
-func (a *App) InstallPlugin(pluginFile io.Reader, replace bool) (*model.Manifest, *model.AppError) {
+func (a *App) InstallPlugin(pluginFile io.ReadSeeker, replace bool) (*model.Manifest, *model.AppError) {
```

Comment:
Nice! 👍  Switched my PR to use `ReadSeeker` as well as opposed to copying the bytes, which would've been expensive.